addons/account_edi/models/account_edi_format.py

```diff
@@ -499,12 +499,21 @@ def _retrieve_partner(self, name=None, phone=None, mail=None, vat=None):
         :param vat:     The vat number of the partner.
         :returns:       A partner or an empty recordset if not found.
         '''
+        normalized_vat = vat
+        if vat and len(vat) > 2:
+            if vat[0].isalpha() and vat[1].isalpha():
+                normalized_vat = vat[2:]
+            try:
+                normalized_vat = str(int(normalized_vat))
+            except ValueError:
+                pass
```

Comment:
Add a comment. Something like "We want to remove 0s at the beginning of normalized_vat if it only consists of digits". Just to make sure people get you're not juste removing 0s in every case (I was tempted to suggest to use lstrip before realising that)